tests/pretty_format_json_test.py

```diff
@@ -105,3 +105,33 @@ def test_top_sorted_get_pretty_format():
 def test_badfile_main():
     ret = main([get_resource_path('ok_yaml.yaml')])
     assert ret == 1
+
+
+def test_diffing_output(capsys):
+    resource_path = get_resource_path('not_pretty_formatted_json.json')
+    expected_retval = 1
+    expected_out = '''\
+--- \n+++ \n@@ -1,6 +1,9 @@
```

Comment:
This is the line I was talking about. `unified_diff` prints:
```
--- 
+++ 
@@ -1,6 +1,9 @@
```
where the first two lines have a trailing space. This was my temporary solution to pass the test cases and satisfy the pre commit check

Reply:
ohh this is where the filename is supposed to be


maybe we should pass in `a/filename` and `b/filename` so that it prints better?

Reply:
where `filename` is the actual filename -- so `os.path.join('a', filename)` and `os.path.join('b', filename)`

Reply:
Ahh that makes sense - maybe we can just put in some pseudo-names? That way it's a little clearer what the output is saying

So it would be
```python
unified_diff(source, target, fromfile='not_pretty.json', tofile='pretty.json')
```

Reply:
yeah, here's an example: https://github.com/asottile/reorder_python_imports/blob/df5d5d28abcf8705bd0c2a622480f6099ca56add/reorder_python_imports.py#L511-L515

Reply:
Gotcha - adding it!